lc-hax/Scripts/Helpers/Players.cs

```diff
@@ -37,4 +37,6 @@ internal static partial class Helper {
             : player;
 
     internal static PlayerControllerB? GetActivePlayer(int playerClientId) => Helper.GetActivePlayer(playerClientId.ToString());
+
+    internal static bool isSelf(this PlayerControllerB instance) => Helper.LocalPlayer != null && instance.actualClientId == Helper.LocalPlayer.actualClientId;
```

Comment:
Methods should be `PascalCase`. This piece of code is used often, so it's worth to optimise it.

```cs
[MethodImpl(MethodImplOptions.AggressiveInlining)]
internal static bool isSelf(this PlayerControllerB instance) => Helper.LocalPlayer is PlayerControllerB localPlayer && instance.actualClientId == localPlayer.actualClientId;
```